src/util/constructRasterAggregatesUrl.js

```diff
@@ -0,0 +1,34 @@
+// import L from "leaflet";
+
+export default function constructRasterAggregatesUrl(raster, latlng) {
+
+    //For non-temporal rasters
+    const layerUrl = '/api/v3/raster-aggregates/?';
+    const aggType = raster.aggregation_type;
+    const srs = 'EPSG:4326';
+    const styles = raster.options.styles;
+    const shortUuid = raster.uuid.substr(0, 7)
+
+    //For temporal rasters, need to include start & stop values
+    //stop value is current date and start value is 2 days ago from current date
+    const stop = new Date();
+    const start = new Date(stop - 2 * 24 * 60 * 60 * 1000);
+
+    //Formate date & time in the format of YYYY-MM-DDTHH:MM:SS
+    const startTime = start.toLocaleDateString().split(/\//).reverse().join('-') + 'T' + start.toLocaleTimeString()
```

Comment:
toLocaleDateString will get the date time in a user format.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString
We probably need a function to get the date time string in a computer format.
Maybe Try to use a UTC format instead.

ALso the timezone offset might be a problem. Does the backend expect the utc global time or user timezone? I think UTC time

Reply:
I fixed the date time format into ISO format. Can you test if it is working now?